CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # App Center SDK for .NET Change Log
 
+## Version 2.5.1 (Under development)
```

Comment:
This feature won't make it to the October release (2.5.0), hence add it to the November release (2.5.1) for now.

Reply:
But this is a feature, so 2.6.0.